### ISSUE TITLE
Linter not showing any issues

### ISSUE THREAD

**Issue by ghost** [AtomLinter/linter-gcc#192]
## Checklist
* [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
* [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
* [x] Ensure that you are using a valid grammar - either C/C++/C++14
* [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):

When copy-pasting the command from the debugging console into terminal, it correctly shows the errors. In Atom no errors are found.

![a](https://user-images.githubusercontent.com/43518060/47615391-837d6d00-daae-11e8-8dbf-24e1bd526986.png)
![b](https://user-images.githubusercontent.com/43518060/47615392-837d6d00-daae-11e8-8985-c6277c15357c.png)
![c](https://user-images.githubusercontent.com/43518060/47615393-837d6d00-daae-11e8-8de5-42c84ce68eac.png)
![d](https://user-images.githubusercontent.com/43518060/47615394-84160380-daae-11e8-9c8c-10cd19a0d882.png)

**Comment by Hex052** [comment]
There was an update in Linter that removed the v1 api, but this package (linter-gcc) does not seem to be maintained anymore. Try @tvincent056's fork, [linter-gcc2](https://atom.io/packages/linter-gcc2). See [comment](https://github.com/hebaishi/linter-gcc/pull/185#issuecomment-435928445) on #185

**Comment by Arcanemagus** [comment]
Duplicate of #199
